assets/js/index.js

```diff
@@ -98,6 +98,24 @@ $(document).ready(function () {
     setTimeout(() => $element.tooltip("hide"), 1500);
   }
 
+  function downloadGithubCssRaw(){
+    const link = "https://raw.githubusercontent.com/shahednasser/sbuttons/master/dist/sbuttons.min.css";
```

Comment:
This is a good approach, however, fetch isn't compatible with all browsers. Please surround `fetch` code with `if (window.fetch)` to check it's supported in the browser and in the else do something like `window.location.href = "https://raw.githubusercontent.com/shahednasser/sbuttons/master/dist/sbuttons.min.css"`  

This way we can support all devices. Also, please replace `const link` with `var link` for browser compatibility as well. Inside the fetch is fine because we already know it will work because of the if condition.  

If there's anything not clear, please let me know.

Reply:
There's also a link that leads to the CSS file in "How to Use" section. Can you also attach the event listener to it as well?

Reply:
Sure, i'll work on that